website/src/pages/index.jsx

```diff
@@ -113,7 +113,7 @@ export default function IndexPage() {
             </h3>
             <p>
               While deck.gl works standalone without a base map, it plays nicely with your favorite
-              base map providers such as Google Maps, Mapbox, ArcGIS and more. Where the base map
+              base map providers such as Google Maps, Mapbox, ArcGIS, MapLibre and more. Where the base map
```

Comment:
Is maplibre just a basemap library, rather than a basemap provider? If so maybe more technically correct...
```suggestion
              base map providers such as Google Maps, Mapbox, ArcGIS, basemap renderers such as MapLibre, and more. Where the base map
```

Reply:
good idea. alternative they could all just be ""base map providers" -> "base maps"